planet/cli/subscriptions.py

```diff
@@ -20,6 +20,10 @@ def _count_fake_subs():
     return len(_fake_subs)
 
 
+def _cancel_fake_sub(sub_id):
+    return _fake_subs.pop(sub_id)
```

Comment:
For our fake API, cancellation means pop an item out of the dictionary that represents the state of the API.